File: bronze/5622/python/main.py

if (__name__ == "__main__"):
    numbers = list(input())
    count = 0
    for data in numbers:
        if (data in ['A', 'B', 'C']):
            count += 3
        if (data in ['D', 'E', 'F']):
            count += 4
        if (data in ['G', 'H', 'I']):
            count += 5
        if (data in ['J', 'K', 'L']):
            count += 6
        if (data in ['M', 'N', 'O']):
            count += 7
        if (data in ['P', 'Q', 'R', 'S']):
            count += 8
        if (data in ['T', 'U', 'V']):
            count += 9
        if (data in ['W', 'X', 'Y', 'Z']):
            count += 10
    print(count)

File: bronze/5622/c++/main.cpp
#include <iostream>
#include <string>

int main()
{
    std::string numbers;
    std::cin >> numbers;
    int count = 0;

    for (auto data : numbers) {
        if (data == 'A' || data == 'B' || data == 'C') {
            count += 3;
        } else if (data == 'D' || data == 'E' || data == 'F') {
            count += 4;
        } else if (data == 'G' || data == 'H' || data == 'I') {
            count += 5;
        } else if (data == 'J' || data == 'K' || data == 'L') {
            count += 6;
        } else if (data == 'M' || data == 'N' || data == 'O') {
            count += 7;
        } else if (data == 'P' || data == 'Q' || data == 'R' || data == 'S') {
            count += 8;
        } else if (data == 'T' || data == 'U' || data == 'V') {
            count += 9;
        } else if (data == 'W' || data == 'X' || data == 'Y' || data == 'Z') {
            count += 10;
        } else {
            return 1;
        }
    }
    std::cout << count;
    
    
    return 0;
}